src/components/shared/nav/index.jsx

```diff
@@ -0,0 +1,186 @@
+    } else {
+      document
+        .getElementsByTagName('body')[0]
+        .classList.remove('u-disable-scroll');
+    }
+  }, [isOpen]);
+
+  console.log({ isOpen, setIsOpen });
+
+  return (
+    <nav className={`nav ${addClass}`}>
+      <Seo />
+      <div
+        className={`nav__content u-content ${addClass} nav__content--desktop`}
+      >
+        <Link to="/">
+          <img alt="nav-logo" className="nav__logo" src={`${logo}`} />
+        </Link>
+        <ul className="nav__menu">
+          <li className="nav__menu__item">
+            <Link to="/"> Home </Link>
+          </li>
+          <li className="nav__menu__item">
+            <Link to="/our-program"> Our Program </Link>
+          </li>
+          <li className="nav__menu__item">
+            <Link to="/apply"> Apply </Link>
+          </li>
+          <li className="nav__menu__item">
+            <Link to="/faq"> FAQ </Link>
+          </li>
+        </ul>
+      </div>
+
+      {isOpen && (
+        <div
+          onClick={() => setIsOpen(false)}
+          style={{
+            position: 'fixed',
+            top: '86px',
+            right: 0,
+            left: 0,
+            bottom: 0,
+            backgroundColor: 'rgba(0,0,0, 0.56)',
+          }}
+        ></div>
+      )}
+      <div
+        className={`nav__content u-content ${addClass} nav__content--mobile`}
+      >
+        <Link to="/">
+          <img alt="nav-logo" className="nav__logo" src={`${logo}`} />
+        </Link>
+        {!isOpen && (
+          <span
+            className="nav__hamburger-icon"
+            onClick={() => setIsOpen(_isOpen => !_isOpen)}
+            style={{ color: menuIconColor }}
+          >
+            <HamburgerIcon />
+          </span>
+        )}
+        {isOpen && (
+          <span
+            className="nav__hamburger-icon"
+            onClick={() => setIsOpen(_isOpen => !_isOpen)}
+            style={{ color: menuIconColor }}
+          >
+            <CloseIcon />
+          </span>
+        )}
+        <ul
+          className={`nav__menu nav__menu--mobile ${
+            isOpen ? 'nav__menu--mobile-open' : ''
+          }`}
+        >
+          <li
+            className="nav__menu__item nav__menu__item--mobile"
+            onClick={() => setIsOpen(false)}
+          >
+            <Link to="/"> Home </Link>
+          </li>
+          <li
+            className="nav__menu__item nav__menu__item--mobile"
+            onClick={() => setIsOpen(false)}
+          >
+            <Link to="/our-program"> Our Program </Link>
+          </li>
+          <li
+            className="nav__menu__item nav__menu__item--mobile"
+            onClick={() => setIsOpen(false)}
+          >
+            <Link to="/apply"> Apply </Link>
+          </li>
+          <li
+            className="nav__menu__item nav__menu__item--mobile"
+            onClick={() => setIsOpen(false)}
+          >
+            <Link to="/faq"> FAQ </Link>
```

Comment:
Is it really necessary to have the menu twice? 
If yes, I understand, why my approaches failed :D

Reply:
I think some css pros can do both menus with the same markup but would be too much effort for me personally to figure it out